components/esp_lvgl_port/test_apps/simd/benchmark/main/test_app_main.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include "unity.h"
+#include "unity_test_utils.h"
+#include "lvgl.h"
+#include "lv_fill_common.h"
+
+#define TEST_MEMORY_LEAK_THRESHOLD (800)
```

Comment:
What would be rational value for the leak check in this case?

Reply:
Some parts of newlib allocate memory after fist call to related functions (e.g. `malloc`, or `fopen`), so you will see some memory leak even though your test freed all allocated memory. Typically memory leak is not produced again in second and subsequent test runs (because newlib allocates the memory only once).

300 bytes is reasonable